### PR TITLE
Add support for starting a preconfigured Firefox instance via `browser.start` command.

### PR DESCRIPTION
**Pull request by sujaldev**
#### Description

`browser.start` command will now allow for a `browser` parameter defaulting to "chrome". (Refer #5247)

#### Checklist

 - [x] I have updated tests where applicable.
 - [x] I have added an entry to the CHANGELOG.

### CODE REVIEW

File: mitmproxy/addons/browser.py [hunk 1]
```diff
@@ -100,6 +100,59 @@ def start(self) -> None:
...
+            return
+
+        host = ctx.options.listen_host or "127.0.0.1"
+        port = ctx.options.listen_port or 8080
+        prefs = [
+            'user_pref("datareporting.policy.firstRunURL", "");',
+            'user_pref("network.proxy.type", 1);',
+            'user_pref("network.proxy.share_proxy_settings", true);',
+        ]
+        for service in ("http", "ssl", "socks", "ftp"):
```
Comment (mhils) [inline mitmproxy/addons/browser.py:129]: `socks` and `ftp` seem wrong here, no?

**Review by mhils — COMMENTED** [review]
Thanks for the PR! 🎉 

1. I get this popup here when running `browser.start firefox`:
    image
    Can we skip that somehow?

2. Are there additional Firefox prefs we can configure to disable telemetry spam / update checks / ...?

File: mitmproxy/addons/browser.py (hunk 1 above)
Comment (sujaldev) [reply mitmproxy/addons/browser.py:129]: sure, I'll remove this.

**Comment by sujaldev** [comment]
> I get this popup here when running browser.start firefox Can we skip that somehow?

This is caused by the `--safe-mode` flag, I'll remove it.

> Are there additional Firefox prefs we can configure to disable telemetry spam / update checks / ...?

I have tried experimenting with a lot of preferences that seem to reduce a tiny amount of telemetry, but requests to `*.services.mozilla.com` and a few other domains just can't seem to be turned off, perhaps we could drop requests to matching domains in mitmproxy?

**Review by mhils — APPROVED** [review]
Awesome, thanks! 🍰 

Good to go from my end - are you still planning any further tweaks, or shall we merge? ☺️

**Comment by sujaldev** [comment]
Thanks for the review! I don't have any more changes planned, so we're good to merge :)

About filtering the remaining telemetry traffic, I think it would just add complexity without much gain, and most users might actually want to see those requests.

**Pull request merged**